ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.tsx

```diff
@@ -245,20 +245,20 @@ export const policyConfigurationDescriptor: Descriptor[] = [
         label: 'Image registry',
         name: 'Image Registry',
         shortName: 'Image registry',
-        longName: 'Image pulled from registry',
-        negatedName: 'Image not pulled from registry',
+        longName: 'Container registry name is',
+        negatedName: 'Container registry name is not',
         category: policyCriteriaCategories.IMAGE_REGISTRY,
         type: 'text',
-        placeholder: 'docker.io',
+        placeholder: 'for example: quay.io',
```

Comment:
For follow up: whether **for example** is relevant elsewhere. For example, next item.